src/services/OutbreakService/OutbreakService.spec.ts

```diff
@@ -330,4 +330,32 @@ describe('OutbreakService', () => {
     const outbreakHistory = service.outbreakHistory.get();
     expect(outbreakHistory).toHaveLength(1);
   });
+
+  it('perform outbreak check', async () => {
```

Comment:
Let's make the test name more descriptive
i.e. 
performs and outbreak check if past the min minutes threshold